src/domain/bridge/contracts/decoders/lifi-diamond-decoder.helper.ts

```diff
@@ -0,0 +1,5694 @@
+import { Injectable } from '@nestjs/common';
+import { AbiDecoder } from '@/domain/contracts/decoders/abi-decoder.helper';
+
+/**
+ * Taken from LI.FI Contract Types:
+ *
+ * @see https://github.com/lifinance/lifi-contract-types/blob/254f29e6eb398e613b6852ca3ac499d00dc586b6/dist/diamond.json
+ *
+ * TODO: We should locate this in @/abis/... but we will need to refactor the /scripts/generate-abis.js
+ * to handle ABIs that are present
+ */
+const LiFiDiamondAbi = [
```

Comment:
This is currently the entire ABI. I suggest we retain only what's necessary after we have finalised transaction decoding.